stripe/src/test/java/com/stripe/android/view/CardInputWidgetTest.java

```diff
@@ -249,17 +246,12 @@ public void onCompleteCardNumber_whenValid_shiftsFocusToExpiryDate() {
         mCardNumberEditText.setText(VALID_VISA_WITH_SPACES);
         assertEquals(R.id.et_card_number, mOnGlobalFocusChangeListener.getOldFocusId());
         assertEquals(R.id.et_expiry_date, mOnGlobalFocusChangeListener.getNewFocusId());
-
```

Comment:
No locking and scrolling. I test the same functionality in the updateSize items later.